src/errorhandling/SDKErrors.ts

```diff
@@ -424,9 +424,9 @@ export const ERROR_DECOMPRESSION_ARRAY: (type?: string) => SDKError = (
 }
 
 export const ERROR_COMPRESS_OBJECT: (
-  object?: object,
+  object?: Record<string, any>,
```

Comment:
should we use `unknown` here? I feel like `unknown` is the cheat to get rid of the warnings! 😁

Reply:
> I feel like `unknown` is the cheat to get rid of the warnings! 😁

Definitely! I started out with `Record<string, unknown>` as suggested for all `object` types. For some cases either the linter or tests broke. For these I changed `unknown` to any. Sometimes these are nested "objects", sometimes arrays and so on. Then I felt like this should be handled in another issue.

Reply:
A ok you can assign any to anything. But unknown needs to be checked. To it's not really a cheat. But yes we should use that and yes that's a different issue.

Reply:
Yes `unknown` seems to be a subset of `any` as it sometimes breaks stuff apart from linter warnings.